src/EnergyPlus/HVACMultiSpeedHeatPump.cc

```diff
@@ -172,6 +172,15 @@ namespace HVACMultiSpeedHeatPump {
         // First time SimMSHeatPump is called, get the input
         if (state.dataHVACMultiSpdHP->GetInputFlag) {
             GetMSHeatPumpInput(state);
+            for (int coilNUM = 1; coilNUM <= int(state.dataHVACMultiSpdHP->MSHeatPump.size()); ++coilNUM) {
+                SetupEMSActuator(state,
+                                 "Coil Speed Control",
+                                 state.dataHVACMultiSpdHP->MSHeatPump(coilNUM).Name,
+                                 "Unitary System DX Coil Speed Level",
+                                 "[]",
+                                 state.dataHVACMultiSpdHP->MSHeatPump(coilNUM).EMSOverrideCoilSpeedNumOn,
+                                 state.dataHVACMultiSpdHP->MSHeatPump(coilNUM).EMSOverrideCoilSpeedNumValue);
+            }
```

Comment:
I looked at DXCoils module. It uses
    if (state.dataGlobal->AnyEnergyManagementSystemInModel) {  
before SetupEMSActuator.

If there are no EMS inputs in an idf file, is it necessary to call this section?